src/GenFacades/Program.cs

```diff
@@ -314,7 +314,7 @@ private static IEnumerable<string> EnumerateDocIdsToForward(IAssembly contractAs
             // Make note that all type forwards (including nested) implement INamespaceAliasForType, so do
```

Comment:
@nguerrera do this comment ring any bells? We hit a bug where the nested type-forwards aren't being copied but the comment seems to indicate that they should.

@joperezr we should also update this comment if we stick with this change.